### PR TITLE
Add custom server routing

### PR DESCRIPTION
**Pull request by JulienPradet**

### CODE REVIEW

File: source/docs/advanced/server/add-http-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+id: overview
+title: Add custom endpoints to your server
+---
+
+The main entrypoints of a Front-Commerce application are frontend URLs displaying your actual website and the GraphQL schema. But in some specific cases, you may want to extend your Node.js server (BFF) with additional endpoints.
```
Comment (real34) [inline source/docs/advanced/server/add-http-endpoint.md]:
```suggestion
The main entrypoints of a Front-Commerce application are frontend URLs displaying your actual website and the GraphQL schema. But in some specific cases, you may want to extend your Node.js server (<abbr title="Backend For Frontend">BFF</abbr>) with additional endpoints.
```

File: source/docs/advanced/server/add-http-endpoint.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    __dangerouslyOverrideBasePathChecks: true,
+    path: "/",
+    router: require("./express")
+  }
+};
+```
+
+Indeed, if you don't set this key to `true`, Front-Commerce will check that you don't setup an invalid path.
+
+For instance, you don't want Front-Commerce to check this if your application is a migration from an older website. Indeed, in this case, you will want to dome some SEO by keeping your old URLs, at least by setting redirections for the ones that don't make sense in your new website. The middleware will let you add this redirection before rendering any page of your website.
```
Comment (real34) [inline source/docs/advanced/server/add-http-endpoint.md]:
```suggestion
For instance, if your application is a migration from an older website, you might want to create a global middleware and would use `__dangerouslyOverrideBasePathChecks`. For SEO reasons, you may want to keep supporting your old URLs and create redirections to relevant locations in your new website. The middleware will let you add this redirection before rendering any page of your website.
```

Comment (JulienPradet) [reply source/docs/advanced/server/add-http-endpoint.md]: That's not clearer IMO. I'll try to find a third alternative.

Comment (real34) [reply source/docs/advanced/server/add-http-endpoint.md]: I agree :) Thanks.

**Comment by JulienPradet** [comment]
Can you please check the new explanation in the latest paragraph?

https://5c7d37c8c06c9d0007941267--elastic-austin-cf9d40.netlify.com/docs/advanced/server/add-http-endpoint.html#Add-a-global-server-middleware

**Pull request merged**